sdk-java/src/main/java/ly/count/sdk/java/Config.java

```diff
@@ -1203,6 +1217,13 @@ public String getCustomDeviceId() {
     public boolean isUsePOST() {
         return usePOST;
     }
+    /**
+     * Getter for {@link #enableBackendMode}
+     * @return {@link #enableBackendMode} value
+     */
+    public boolean isBackendModeEnable() {
```

Comment:
Remove this call, it is not needed. Just access the value directly

Reply:
I  have to make it public to access.